docs/svelte-testing-library/setup.mdx

```diff
@@ -4,85 +4,83 @@ title: Setup
-    ```
+1. Install Vitest and jsdom
 
-2.  Add the following to your `package.json`
+We're using `jdom` here as the test environment, but you can use any other
+options e.g `happy-dom`.
 
-    ```json
-    {
-      "scripts": {
-        "test": "vitest run src",
-        "test:watch": "vitest src"
-      }
-    }
-    ```
+```bash npm2yarn
+npm install --save-dev vitest jsdom
+```
 
-3.  You'll need to compile the Svelte components before using them in Vitest, so
-    you need to install
-    [@sveltejs/vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte) and Vite
-    
+2. Add the test scipts to your `package.json` to run the tests with Vitest
 
-    ```
-    npm install --save-dev @sveltejs/vite-plugin-svelte vite
-    ```
+   ```json
+   {
+     "scripts": {
+       "test": "vitest run",
+       "test:ui": "vitest --ui",
+       "test:watch": "vitest"
+     }
+   }
+   ```
 
-4.  Add a `vitest.config.ts` configuration file to the root of your project
+3. To compile the Svelte components before using them in Vitest, you need to
+   install
+   [@sveltejs/vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte)
+   and Vite
 
-    ```js
-    import { defineConfig } from 'vitest/config';
-    import { svelte } from '@sveltejs/vite-plugin-svelte';
+   ```bash npm2yarn
+   npm install --save-dev @sveltejs/vite-plugin-svelte vite
+   ```
 
-    export default defineConfig({
-        plugins: [svelte({ hot: !process.env.VITEST })],
-        test: {
-            include: ['src/**/*.{test,spec}.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-            globals: true,
-            environment: 'jsdom'
-        }
-    });
+4. Add a `vitest.config.ts` configuration file to the root of your project
 
-    ```
+   ```js
+   import {defineConfig} from 'vite'
+   import {svelte} from '@sveltejs/vite-plugin-svelte'
 
-5.  This is optional but it is recommended, you can install
-    [jest-dom](https://github.com/testing-library/jest-dom) to add handy
-    assertions to Jest
+   export default defineConfig({
+     plugins: [svelte({hot: !process.env.VITEST})],
+     test: {
+       globals: true,
+       environment: 'jsdom',
+     },
+   })
+   ```
 
-    5.1 Install `jest-dom`
+5. Optionally install [jest-dom](https://github.com/testing-library/jest-dom) to
+   add handy assertions to Vitest
 
-    ```
-    npm install --save-dev @testing-library/jest-dom
-    ```
+   5.1 Install `jest-dom`
 
-    5.2 import `@testing-library/jest-dom` at the start of your test files
+   ```bash npm2yarn
+   npm install --save-dev @testing-library/jest-dom
+   ```
 
-    ```js
-    import '@testing-library/jest-dom';
-    ```
+   5.2 import `@testing-library/jest-dom` at the start of your test files
 
-6.  Create your component and a test file (checkout the rest of the docs to see how)
-    and run the following command to run the tests.
+   ```js
+   import '@testing-library/jest-dom'
+   ```
 
-    ```
-    npm run test
-    ```  
-### SvelteKit
```

Comment:
Removed this because the rest of the sections are about setting up test runners.
I added a sveltekit section to the FAQ.
With the current state, everything works per default with SvelteKit.
Thus, the steps that were added to install `vitest-svelte-kit` can be removed.